cob/cob.c

```diff
@@ -1708,34 +1752,41 @@ M0_INTERNAL int m0_cob_name_update(struct m0_cob *cob,
 	struct m0_buf        val;
 	int                  rc;
 
+	uint16_t            ht_srcidx;
+	uint16_t            ht_tgtidx;
+	uint16_t            ht_idx;
```

Comment:
These variables are used only once.
1. Can we use a single variable?
2. Can we:
```
cob_table_lookup(&cob->co_dom->cd_namespace[cob_get_hash(&srckey->cnk_pfid)], &key, &val);
```